src/splitInteger.test.js

```diff
@@ -1,17 +1,17 @@
 const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(3, 1)).toEqual([3]);
 });
 
```

Comment:
it's better to use  `expect(splitInteger(6, 2)).toEqual([3, 3]);` assertion for this case, because you have only one value as a result but you should have some equal parts